github-pullrequest-plugin/src/main/resources/com/github/kostyasha/github/integration/branch/GitHubBranchRepository/index.groovy

```diff
@@ -55,9 +65,10 @@ l.layout(title: "GitHub Branch Status") {
                 if (h.hasPermission(Item.BUILD)) {
                     tr() {
                         td() {
-                            def rebuildId = "rebuildResult" + branch.name;
+                            def rebuildId = makeRebuildId(branch.name);
+                            def escaped = escapeBranchName(branch.name);
```

Comment:
static import and
`def escaped = escapeEcmaScript(branch.name)` and groovy method not needed.